# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/moz.build
# Commit: fef894fcc429
# Full Hash: fef894fcc4294027fdfb62af3997b87411bef554
# Author: Jan Grulich <jgrulich@redhat.com>
# Date: 2023-11-30 17:23:44
# Regressor Bug: 1844020
# File Overlap Count: 1
# Description:
#   Bug 1844020 - Add a camera device placeholder in case PipeWire is not initialized yet r=pehrsons
#   
#   Implement DeviceInfoPipeWire with option to add a device placeholder
#   when there is a camera device, but we don't have access to PipeWire yet.
#   We ask the Camera portal whether there is a camera device present.
# ==============================================================================

diff -r ed31b2acb5fb -r fef894fcc429 dom/media/systemservices/moz.build
--- a/dom/media/systemservices/moz.build	Thu Nov 30 11:49:35 2023 +0000
+++ b/dom/media/systemservices/moz.build	Thu Nov 30 11:49:36 2023 +0000
@@ -58,6 +58,9 @@
         ]
 
     if "WEBRTC_USE_PIPEWIRE" in DEFINES:
+        UNIFIED_SOURCES += [
+            "video_engine/placeholder_device_info.cc",
+        ]
         CXXFLAGS += CONFIG["MOZ_GTK3_CFLAGS"]
 
 if CONFIG["OS_TARGET"] == "Android":